lib/asciidoctor/js/asciidoctor_ext/browser/abstract_node.rb

```diff
@@ -1,5 +1,29 @@
 module Asciidoctor
 class AbstractNode
+  def read_contents target, opts = {}
+    doc = @document
+    if (Helpers.uriish? target) || ((start = opts[:start]) && (Helpers.uriish? start) && (target = doc.path_resolver.web_path target, start))
+      if (doc.path_resolver.descends_from? target, doc.base_dir) || (doc.attr? 'allow-uri-read')
```

Comment:
In a browser environment we should allow to read URI if the path is descended from the base directory (even if the `allow-uri-read` attribute is not defined)

Reply:
Agreed. I think we made a similar change in the Reader class.

Reply:
Exactly I've copied the line from the Reader class :smile: